interfaces/offscreen_effect_player.hpp

```diff
@@ -46,8 +46,9 @@ namespace bnb::oep::interfaces
          * @param target_orientation image orientation for postprocessing
          *
          * @example process_image_async(my_input_image, rotation::deg0, true, [](image_processing_result_sptr sptr){}, rotation::deg180)
+         * @return false if the frame was dropped, otherwise true
```

Comment:
return false if the frame is rejected because of too many items in the internal queue of frames, otherwise true